.github/workflows/build-image.yml

```diff
@@ -0,0 +1,106 @@
+name: Build Container Image
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '39 11 * * *'
```

Comment:
why is this a cron job? to keep it on the docker registry site? a comment would be nice here

Reply:
That was there from GitHub's default action, I'll remove it.